proto/pdpb.proto

```diff
@@ -163,10 +160,9 @@ enum ErrorType {
     ALREADY_BOOTSTRAPPED = 4;
     INCOMPATIBLE_VERSION = 5;
     REGION_NOT_FOUND = 6;
-    GLOBAL_CONFIG_NOT_FOUND = 7;
-    DUPLICATED_ENTRY = 8;
-    ENTRY_NOT_FOUND = 9;
-    INVALID_VALUE = 10;
+    DUPLICATED_ENTRY = 7;
```

Comment:
```suggestion
    reserved 7;
```
We should just deprecate this variant but keep other variants unchanged so we can be compatible with old versions.

Reply:
get it, Done.